### PR TITLE
Updated Event Agenda Display

### PR DESCRIPTION
**Pull request by thecristen**
#### Summary of changes
**Asana Ticket:** [[Elixir] Updated agenda display (CSS/HTML)](https://app.asana.com/0/385363666817452/1200184323508002/f)

Building upon @hannahpurcell's initial work, makes the agenda all pretty and stuff.

image

---

Before getting review, please check the following:

* [x] Does frontend functionality render and work correctly in IE?
* [x] Are the changes organized into self-contained commits with descriptive and well-formatted commit messages?

### CODE REVIEW

File: apps/site/assets/css/_event-details.scss [hunk 1]
```diff
@@ -41,10 +43,129 @@
...
+    }
+
+    counter-increment: $counter;
+    display: inline-block;
+    font-weight: 700;
+    height: $bullet-size;
+    line-height: $bullet-size;
+    text-align: center;
+    width: $bullet-size;
+  }
```
Comment (alicia-mbta) [inline apps/site/assets/css/_event-details.scss:79]: I love this and its reusability 💯

File: apps/site/assets/.stylelintrc [hunk 2]
```diff
@@ -39,6 +39,7 @@
       2,
       {
         "ignore": [
+          "param",
```
Comment (alicia-mbta) [inline apps/site/assets/.stylelintrc:42]: What does this do?

File: apps/site/lib/site_web/controllers/customer_support_controller.ex [hunk 3]
```diff
@@ -140,8 +140,7 @@ defmodule SiteWeb.CustomerSupportController do
 
   @spec do_submit(Plug.Conn.t(), map) :: Plug.Conn.t()
   defp do_submit(%Plug.Conn{assigns: %{ip_address: {:ok, ip}}} = conn, data) do
-    rate_limit_interval =
-      if Application.get_env(:feedback, :rate_limit, true), do: 60_000, else: 1_000
+    rate_limit_interval = Application.get_env(:feedback, :rate_limit, 60_000)
```
Comment (alicia-mbta) [inline apps/site/lib/site_web/controllers/customer_support_controller.ex:143]: Is this to avoid timeouts when submitting the form (in real life and in the tests)?

**Review by alicia-mbta — APPROVED** [review]
Just a few questions (for me to understand) but otherwise it looks so good!

File: apps/site/lib/site_web/controllers/customer_support_controller.ex (hunk 3 above)
Comment (thecristen) [reply apps/site/lib/site_web/controllers/customer_support_controller.ex:143]: It leaves the default rate limit at 60s (so if someone tries to submit the form more than once a minute, they get stopped). Using the env variable lets me override that, which I do for tests (where many tests running very quickly sometimes in parallel will very easily hit the rate limit! so in the config file I set this value to 1s instead)

File: apps/site/assets/.stylelintrc (hunk 2 above)
Comment (thecristen) [reply apps/site/assets/.stylelintrc:42]: https://stylelint.io/user-guide/rules/list/indentation/#ignore-inside-parens-param-value

Allows arbitrary indentation of function params. 
TLDR Lets me write this without complaining haha

```scss
@mixin agenda-bullet($bg-color: $gray, $color: $white,
                     $counter, $counter-type: false)
```

File: apps/site/lib/site_web/controllers/customer_support_controller.ex (hunk 3 above)
Comment (thecristen) [reply apps/site/lib/site_web/controllers/customer_support_controller.ex:143]: @amaisano Hm yeah, that was pretty out of scope of me 🤔
